### DISCUSSION TITLE
Welcome to Git for Windows' Discussions!

### DISCUSSION THREAD

**Discussion by dscho** [git-for-windows/git#2929] [General]
## Welcome to Git for Windows!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Contribute using your talents to make Git for Windows better.

To keep this a safe space, let's always be respectful and be open to learn from each other.

**Comment by PhilipOakley** [comment]
Hi Dscho,

I was wondering what "Discussions" was, and had left your welcome email bookmarked to come back to. 

If I understand correctly "Discussions" is a type of tab on the GitHub 'Git for Windows' project to complement the "Issues" and "Code" tabs (it's the next one after the "Pull Request" tab, just before the "Actions". And then the "Wiki" tab is a bit further to the right.

One thing that could be discussed is how to present the flatness of the project structure (if we can call the people organisation that..). It feels that many folks out there think that there must be some mega team of ninja coders ready to leap at any suggestion or issue, in "the Windows Way" (is that a thing?), when the reality is far less 'organised'. I'm a bit loathe to open the potential can of worms of such a discussion as they often just need a bigger can to hold the discussion! Are there are any ideas (Question to one and all) about how to simplify the engagement of folks into the project?


Also, is this Discussions tab a catch all topic, or can new ones be started? I see that is #2929 already - is it really a variant of the Issues bit with some hidden switch flicked. How's that (starting a new discussion topic) done?

Hopefully this reply at least starts some others thinking about dropping in for a chat, (given half the world looks to be in lockdown of some sort)

**Reply by dscho** [reply]
Personally, I am curious to see in what ways people use the Discussions feature, and don't want to impose any rules myself (instead, I'd like to keep the rules more consensus-based).

It's kind of neat that issues can be converted to discussions when it turns out that the issue isn't about a bug in Git for Windows at all.

**Reply by PhilipOakley** [reply]
I hadn't even realised there was a discussion feature! (hence the follow up of your notification..)

I see that the notification emails don't indicate that it's a discussion or has change to a discussion...

Maybe someone (of the readers) could add a link on the home-page and maybe the FAQs

**Reply by derrickstolee** [reply]
Discussions are mostly a way to separate user requests and questions from the project's TODO list (issues).

The threaded sub-conversations (like this one) help with this.

Also, we _could_ use it as a replacement for StackOverflow, except the maintainer has more control over which answers are "accepted", or could remove answers that are misleading or incorrect. I'm interested to see if users start coming here for that purpose.

**Reply by radnin-tYgxy0-deggef** [reply]
I don’t know what the issue with this is but I’m trying not to think that it’s a big problem because it’s just the fact that the people who have been in contact with you for years have had a lot of issues with the same issue that I have had with the same problem that I’ve been dealing with for years now but it’s just the way things have been with the last few years I have been trying to figure it all out and it’s been very frustrating to see how much of the work has gone on with the people that I have dealt with and I have been in contact with but I have had a problem with the way that I have been dealing with the people who have not been in the past but it’s not just a little more than a lot more of the same situation I have been trying to figure out what is really frustrating to me is the fact that you have been doing this and it’s not helping me to get the same problem with the way things have been very difficult and that’s why I don’t want to get the other people that is the way it was the problem is that you know how much of the situation I have been through this whole time it’s been very hard to get over the past few years I’ve tried so much to help you but you have a lot more problems with your life and you don’t even talk about it so you just keep trying to help yourself and I know that I don’t want you and I do I know you’re a good friend and you have been through a lot but I’m just not sure what to tell you about this situation

**Comment by EslamGood** [comment]
> ## Welcome to Git for Windows!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Contribute using your talents to make Git for Windows better.
> 
> To keep this a safe space, let's always be respectful and be open to learn from each other.
